docs/kubernetes-compatibility.md

```diff
@@ -0,0 +1,53 @@
+# Kubernetes Version Compatibility
+
+Kubernetes supports 3 minor release concurrently. This means if the latest
+Kubernetes release is v1.19, that v1.18 and v0.17 will remain supported for
```

Comment:
I think that you meant v1.17 and v1.16 and not 0.X

Reply:
ha, oops. thanks